api/src/config/aws/creds.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * This module configures and exports an AWS Credentials object that can be reused.
+ */
+const {defaultProvider} = require('@aws-sdk/credential-provider-node');
+
+// reads from env file for AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_SESSION_TOKEN (if given)
+const credentials = defaultProvider();
```

Comment:
Could this be also done using `dotenv` and using the default AWS Credentials object?

Reply:
I couldn't get it to work with what you had, but I also upgraded to use v3 of the aws sdk, which doesn't have the credentials object so I went with this instead. see: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/modules/_aws_sdk_credential_provider_node.html

Reply:
LGTM